packages/localization/src/scripts/UntranslatedStrings.js

```diff
@@ -0,0 +1,50 @@
+// @flow strict
+
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+
+import { getVocabularies } from '../DefaultVocabulary';
+import {
+  _fetch,
+  _paginate,
+  findKeys,
+  _log,
+  _removePrefix,
+} from './PhraseAppHelpers';
+
+const localTranslations = getVocabularies([
```

Comment:
Maybe add the same comment as here https://github.com/kiwicom/mobile/pull/1504/files#diff-74c59ca94d3d71b23941003d7840e88eL21

or extract that to a common file too?

Reply:
Sure 👍

Reply:
🚀